pkg/pao/pao.go

```diff
@@ -0,0 +1,70 @@
+package pao
```

Comment:
Did you mean to leave this file in? If so, can we make updates so that it adheres to the standard builder definitions and factories that we have in other pkgs? Please reference other pkgs for examples